### PR TITLE
[11.x] Add Controller & Request to structure

### PR DESCRIPTION
**Pull request by milwad-dev**
https://github.com/laravel/docs/pull/9755

### CODE REVIEW

File: structure.md [hunk 1]
```diff
@@ -137,6 +137,16 @@ The `Exceptions` directory contains all of the custom exceptions for your applic
 
 The `Http` directory contains your controllers, middleware, and form requests. Almost all of the logic to handle requests entering your application will be placed in this directory.
 
+<a name="the-controller-directory"></a>
+#### The Controller Directory
+
+The `Controller` directory exists in Http folder and contains your [controllers](https://laravel.com/docs/controllers). All controllers generated by the `make:controller` command exist in this folder.
```
Comment (siarheipashkevich) [inline structure.md:143]: `Controllers` directory maybe?

File: structure.md [hunk 2]
```diff
@@ -137,6 +137,16 @@ The `Exceptions` directory contains all of the custom exceptions for your applic
...
+
+<a name="the-request-directory"></a>
+#### The Request Directory
+
+The `Request` directory exists in Http folder and contains your [requests](https://laravel.com/docs/requests). All requests generated by the `make:request` command exist in this folder.
```
Comment (siarheipashkevich) [inline structure.md:148]: `Requests` directory maybe?

File: structure.md (hunk 1 above)
Comment (milwad-dev) [reply structure.md]: 👍

**Comment by taylorotwell** [comment]
It's fine.

**Pull request closed without merging**